Packs/Gurucul/ReleaseNotes/1_1_3.md

```diff
@@ -0,0 +1,22 @@
+#### Incident Fields
+- Added Gra Weblink
+
+#### Integrations
+##### Gurucul-GRA
+
+- **Breaking Change**: New drop-down ***Fetch Incident By Case Status*** is added in ***Add instance***. The valid options are:
```

Comment:
Why is this a breaking change? the parameter is not mandatory

Reply:
Removed breaking change and commited code